uportal-war/src/main/java/org/jasig/portal/layout/dlm/remoting/ChannelListController.java

```diff
@@ -79,42 +82,50 @@
 	private IPortalSpELService spELService;
 	private ILocaleStore localeStore;
 	private MessageSource messageSource;
-	private static final String TYPE_SUBSCRIBE = "subscribe";
+    private IAuthorizationService authorizationService;
+
+    private static final String UNCATEGORIZED = "uncategorized";
+    private static final String UNCATEGORIZED_DESC = "uncategorized.description";
 
     /**
      * @deprecated Moved to PortletRESTController under /api/portlets.json
      */
     private static final String TYPE_MANAGE = "manage";
 
-	/**
+    /**
+     * @deprecated in uPortal 4.2.  Rather than using subscribe and categories <> null, returns portlets that user
+     * has BROWSE permission to regardless whether the portlet has a category to be consistent with Marketplace,
+     * Search, and the BROWSE permission.
+     */
+    private static final String TYPE_SUBSCRIBE = "subscribe";
+
+    /**
 	 * 
 	 * @param request
 	 * @param type
-	 * @param asXml
 	 * @return
 	 */
 	@RequestMapping(method = RequestMethod.GET)
 	public ModelAndView listChannels(WebRequest webRequest, HttpServletRequest request, @RequestParam(value="type",required=false) String type) {
-        type = type != null ? type : TYPE_SUBSCRIBE;
-        if(TYPE_MANAGE.equals(type)) {
+        if(type != null && TYPE_MANAGE.equals(type)) {
             throw new UnsupportedOperationException("Moved to PortletRESTController under /api/portlets.json");
         }
```

Comment:
All was removed in 4/15/10 for UP-2672.  It was so far in the past I just removed it since no one would have been using it at this point.